source/Engine.cpp

```diff
@@ -1181,6 +1230,16 @@ void Engine::Draw() const
 	draw[currentDrawBuffer].Draw();
 	batchDraw[currentDrawBuffer].Draw();
 
+	for(const Gunsight &gunsight : gunsights)
+	{
+		Point end1 = gunsight.start + gunsight.angle.Rotate(Point(gunsight.spread, -gunsight.range));
+		Point end2 = gunsight.start + gunsight.angle.Rotate(Point(-gunsight.spread, -gunsight.range));
+		Point width = gunsight.angle.Rotate(Point(1., 0.));
+
+		LineShader::DrawGradient(gunsight.start * zoom, end1 * zoom, width.Length(), gunsight.color, Color(0, 0, 0, 0));
```

Comment:
What is width supposed to do here? Is it not just `1.`?

You are rotating a vector of length one which gives a vector of length one, then taking the length of that vector (again, one).